portfolio/src/main/webapp/script.js

```diff
@@ -83,3 +83,11 @@ function refreshComments(num) {
     });
   });
 }
+
+/**
+ * Use POST request with fetch to delete all comments
+ */
+function deleteComments() {
+    const request = new Request('/delete-data', {method: 'POST'});
+    fetch(request).then(response => response.json()).then(refreshComments(0));
```

Comment:
Why 0?

Reply:
I put a random number in there, and since it's going to all be deleted anyways, I chose 0. Should I use another number?

Reply:
Ah ok, just realized that you're deleting all comments, but I think delete comment should pass the comment key to the server and only delete that particular comment.

Reply:
The tutorial asked us to delete all the comments. I'm going to try to implement that though once I get past the requirements!

Reply:
Got it, sorry I didn't read the tutorial carefully :(